website/content/docs/api-clients/go-sdk.mdx

```diff
@@ -99,36 +95,43 @@ Lastly, let's update the original client with the token we got from the `Authent
 
 ```go
 // pass this client to any other resource specific API resources
-client.SetToken(at.Item.Token)
+client.SetToken(fmt.Sprint(authenticationResult.Attributes["token"]))
 ```
 
 Putting this all together:
 
 ```go
+package main
```

Comment:
I don't think this should be structured as a complete program -- it's not. It doesn't do anything useful, and it's not fully structured -- `main` doesn't even return. The example is meant as integration advice within a program.

Reply:
Makes sense.  I will update to reflect that goal.